salt/rvm/init.sls

```diff
@@ -0,0 +1,32 @@
+{% from 'lib.sls' import create_user %}
+
+rvm:
+  pkgrepo.managed:
+    - ppa: rael-gc/rvm
+  pkg.installed:
+    - name: rvm
+    - require:
+      - pkgrepo: rvm
+
+# A Salt bug breaks support for the apt-managed package.
```

Comment:
Is this bug documented anywhere?

Reply:
This bug of it not supporting this path to `rvm` hasn't been raised with Salt yet. We should raise it and link in the code.

Reply:
I have been working on a PR to fix this directly in Salt, no need to delay the Redmine PR while I work on this.
I'll put a comment in salt when there is a GitHub issue / PR to link to.

Reply:
Bug report linked.